pkg/translator/testdata/refinery_rules/sampleerrors_defaults.yaml

```diff
@@ -0,0 +1,25 @@
+RulesVersion: 2
+Samplers:
+    __default__:
+        RulesBasedSampler:
+            Rules:
+                - Conditions:
+                    - Datatype: int
+                      Fields:
+                        - http.status_code
+                        - http.response.status_code
+                      Operator: '>='
+                      Value: "500"
+                  Name: Sample 500 statuses at 1
+                  SampleRate: "1"
+                - Conditions:
+                    - Datatype: int
+                      Fields:
+                        - http.status_code
+                        - http.response.status_code
+                      Operator: '>='
+                      Value: "400"
+                  Name: Sample 400 statuses at <no value>
```

Comment:
I think the name has got a little messed up here - should be `Sample 400 statuses at 10`. Same happened for Sample Remainder below, but not for 500s.

Reply:
Good catch.